src/ir/ir.jl

```diff
@@ -173,6 +297,12 @@ end
 
 explicitbranch!(ir::IR) = (foreach(explicitbranch!, blocks(ir)); return ir)
 
+"""
+    branches(b::Block, c::Block)
+    branches(b::Block, c::Integer)
```

Comment:
I think we should remove this `Block, Int` method(from the docs at least, and perhaps also entirely); its signature does not seem very clear to me.

Reply:
I left it in the code, since there are apparently passes depending on it, but removed the line from the docstring.